src/store/listAnime.js

```diff
@@ -330,6 +349,12 @@ export default [
     animeYear: 2015,
     animeLink: 'https://myanimelist.net/anime/30276/One_Punch_Man',
   },
+  {
```

Comment:
one puchman is already added 
please delete this

Reply:
i already delete it